docs/using/docker-image.rst

```diff
@@ -136,6 +145,7 @@ After pushing your image to your Docker image registry, you can do the deploy us
 
     This image should be in a registry and be accessible by the nodes.
     Image should also have a Entrypoint or a Procfile at given paths, / or /app/user/ or /home/application/current
+    The Image should not expose a Port! This is done automatically using the $PORT environment variable.
```

Comment:
You can remove this sentence, as you already explained before that the image could expose a port or use the $PORT variable.